prow/jobs/incubator/compass/tests/provisioner-tests/provisioner-whitesource.yaml

```diff
@@ -0,0 +1,54 @@
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+
+whitesource_job_template: &whitesource_job_template
+  skip_report: false
+  run_if_changed: "go.mod|go.sum"
+  decorate: true
+  optional: true
+  path_alias: github.com/kyma-incubator/compass/components/provisioner
+  max_concurrency: 2
```

Comment:
The concurrency should be `1`, as whitesource will override results and there is no reason in that case to have two running at the same time.